full/src/test/java/apoc/custom/CypherProceduresTest.java

```diff
@@ -114,6 +194,13 @@ public void registerSimpleStatementFunction() throws Exception {
         TestUtil.testCall(db, "return custom.answer2() as row", (row) -> assertEquals(42L, row.get("row")));
     }
 
+    @Test
+    public void registerSimpleStatementFunctionWithOneChar() throws Exception {
+        db.executeTransactionally("call apoc.custom.asFunction('a','RETURN 42 as answer')");
+        TestUtil.testCall(db, "return custom.a() as row", (row) -> assertEquals(42L, ((Map)((List)row.get("row")).get(0)).get("answer")));
+        TestUtil.testFail(db, "CALL apoc.custom.declareFunction('b() :: STRING','RETURN 42 as answer')", QueryExecutionException.class);
```

Comment:
might be useful to test the error message

Reply:
Changed